code/src/components/Scarpariello.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+export const Scarpariello = () => {
+  return (
+    <>
+      <div className="header">
+        <h1 className="heading">Scarpiarello</h1>
+      </div>
+      <div className="recipe-container">
+        <div className="recipe-box">
+          <div className="recipe-content">
+            <div className="ingredients">
```

Comment:
Great that every recipe has the same structure with recipe-container, box, content and ingredients, it'll save time in CSS.